File: C PROGRAMMING LANGUAGE/ASSIGNMENTS/ASSIGNMENT-8/sum.c
int sum(int[] ,int);
void main()
{
	int arr[100],size;
	printf("enter size of array: ");
	scanf("%d",&size);
	printf("\nenter array elements: ");
	for(int i=0;i<size;i++)
		scanf("%d",&arr[i]);
	printf("\narray elements: ");
	for(int i=0;i<size;i++)
		printf("%d  ",arr[i]);
	printf("\n%d is sum of array elements",sum(arr,size));
}
int sum(int a[],int n)
{
	int sum=0;
	for(int i=0;i<n;i++)
	{
		sum+=a[i];
	}
	return sum;
}

File: JAVA/ARRAY/Sum.java
//3.	Find sum of all numbers.
import java.util.Scanner;

class Sum
{
  public static void main(String[] args)
  {

	Scanner sc = new Scanner(System.in);
	int i=0,sum=0;
	int [] arr = new int[10];
	System.out.println("enter the array elements: ");
	for(i=0;i<arr.length;i++)
	{
	   arr[i]=sc.nextInt();
 	}
	for(i=0;i<arr.length;i++)
	{
	   sum +=arr[i];
 	}


	System.out.println("Sum of elements is : "+sum);
 
  }
}